specs/core/0_beacon-chain.md

```diff
@@ -678,7 +679,7 @@ def deposit(deposit_input: bytes[2048]):
         self.deposit_tree[index] = sha3(concat(self.deposit_tree[index * 2], self.deposit_tree[index * 2 + 1]))
 
     self.deposit_count += 1
-    if msg.value == MAX_DEPOSIT_AMOUNT:
+    if msg.value == as_wei_value(MAX_DEPOSIT, "gwei"):
```

Comment:
this should be `as_wei_value(MAX_DEPOSIT_AMOUNT, "gwei")`.

nice find on the `as_wei_value` functionality! wasn't sure if we could do a Gwei basis in vyper or not. looks like we can :)